ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -124,7 +124,7 @@ final class CommandsValidator(ledgerId: LedgerId) {
       } yield validatedInnerCommands :+ validatedInnerCommand
     })
 
-  private def validateInnerCommand(
+  def validateInnerCommand(
```

Comment:
why do you need to open up both? it would be nice if we could keep cross-repo dependencies small.

Reply:
In this case, we have two different representations of the same concept (ledger api command and DAMLe command). Our life gets a lot easier, if the conversion functions are public and easy to find.